test/proto3pb/test_all_types.proto

```diff
@@ -94,6 +99,9 @@ message TestAllTypes {
   // Map
   map<string, string> map_string_string = 58;
   map<int64, NestedTestAllTypes> map_int64_nested_type = 59;
+
+  // Imports
+  repeated ImportedGlobalEnum imported_enums = 60;
```

Comment:
Can you make the corresponding updates in cel-spec?

Reply:
Sure.